etc/config-template.toml

```diff
@@ -798,7 +786,6 @@
 # info-log-keep-log-file-num = 10
 # info-log-dir = ""
 # info-log-level = "info"
-# optimize-filters-for-hits = true
```

Comment:
Not present anymore in `RaftDbConfig` 
https://github.com/tikv/tikv/blob/master/src/config.rs#L1264